lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -969,8 +971,20 @@ data ByronWalletPutPassphraseData = ByronWalletPutPassphraseData
     , newPassphrase :: !(ApiT (Passphrase "user"))
     } deriving (Eq, Generic, Show)
 
+data ApiSealedTxEncoding = HexEncoded | Base64Encoded
+      deriving (Eq, Generic, Show)
+      deriving anyclass NFData
+
+instance ToJSON ApiSealedTxEncoding where
+    toJSON HexEncoded = Bool True
+    toJSON Base64Encoded = Bool False
```

Comment:
This instance means that `ApiSealedTxEncoding` is represented as `Bool` in the JSON. 
I'd suggest to represent it as `String` with 2 possible values "hex" and "base64". OpenAPI supports `enum` which is convenient in case like this one.

Reply:
sounds like a good idea - done